03/app.js

```diff
@@ -6,28 +6,55 @@ const root = createRoot(document.querySelector('#root'));
 class Article extends React.Component {
     state = {
         comments: [],
+        text:'',
     }
+
+    addComments () {
+        return this.state.comments.map (comm =>{
+            return <li>{comm}</li>
+        })
+    }
+
+    handleSubmit = e => {
+        e.preventDefault();
+
+        this.setState({
+            comments: {...this.state.comments, text},
```

Comment:
Nie ma takiej zmiennej jak `text` - zadanie nie działa :P

PS. Jacku, musisz sprawdzać rozwiązania przed oddaniem. Jeśli na rekrutacji oddasz niedziałające zadanie to nikt nie zajrzy nawet do kodu, tylko od razu Cię skreśli :/